src/towncrier/_settings/load.py

```diff
@@ -29,25 +28,25 @@
     import tomllib
 
 
-@dataclass
+@dataclasses.dataclass
 class Config:
-    package: str
-    package_dir: str
-    single_file: bool
-    filename: str
-    directory: str | None
-    version: str | None
-    name: str | None
     sections: Mapping[str, str]
     types: Mapping[str, Mapping[str, Any]]
     template: str
-    start_string: str
-    title_format: str | Literal[False]
-    issue_format: str | None
-    underlines: list[str]
-    wrap: bool
-    all_bullets: bool
-    orphan_prefix: str
+    package: str = ""
+    package_dir: str = "."
+    single_file: bool = True
+    filename: str = "NEWS.rst"
+    directory: str | None = None
+    version: str | None = None
+    name: str = ""
+    start_string: str = ".. towncrier release notes start\n"
```

Comment:
I have just realized, that this makes no sense with Markdown (you'd have to use an HTML comment like `<!-- towncrier release notes start -->\n`) and that we don't have a test for the case where an existing news file is amended – correct?

If things are supposed to work out of the box, I guess you'll have to special-case this too (and add a test for it).